configure.ml

```diff
@@ -647,8 +647,10 @@ let camltag = match caml_version_list with
     48: implicit elimination of optional arguments: too common
     50: unexpected documentation comment: too common and annoying to avoid
     56: unreachable match case: the [_ -> .] syntax doesn't exist in 4.02.3
+    59: "no cmx file was found in path" See https://github.com/ocaml/num/issues/9
+    59: "A potential assignment to a non-mutable value was detected in this source file." See issue #8043
```

Comment:
These comments are a bit messed up.

Reply:
Let's see now.

Reply:
Still messed up: the commit for -59 is adding a duplicated comment.

Reply:
should be fixed now.